docs/source/notebooks/tutorial.ipynb

```diff
@@ -278,30 +272,156 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "It is possible to interact with the `DataFrame`; for instance, we can see what the \"attributes\" of the datasets are by printing the columns."
+    "## Catalog Entries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "3547"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "len(col.keys())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Intake ESM uses information saved in the `key_template` property in order to figure out how to create catalog entry keys:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 5,
    "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "activity_id.institution_id.source_id.experiment_id.table_id.grid_label\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(col.key_template)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's take a look at the first ten entries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "Index(['activity_id', 'institution_id', 'source_id', 'experiment_id',\n",
-       "       'member_id', 'table_id', 'variable_id', 'grid_label', 'zstore',\n",
-       "       'dcpp_init_year'],\n",
-       "      dtype='object')"
+       "['AerChemMIP.AS-RCEC.TaiESM1.histSST.AERmon.gn',\n",
+       " 'AerChemMIP.BCC.BCC-ESM1.histSST.AERmon.gn',\n",
+       " 'AerChemMIP.BCC.BCC-ESM1.histSST.Amon.gn',\n",
+       " 'AerChemMIP.BCC.BCC-ESM1.piClim-CH4.Amon.gn',\n",
+       " 'AerChemMIP.BCC.BCC-ESM1.piClim-NTCF.Amon.gn',\n",
+       " 'AerChemMIP.BCC.BCC-ESM1.piClim-control.AERmon.gn',\n",
+       " 'AerChemMIP.BCC.BCC-ESM1.piClim-control.Amon.gn',\n",
+       " 'AerChemMIP.BCC.BCC-ESM1.ssp370.AERmon.gn',\n",
+       " 'AerChemMIP.BCC.BCC-ESM1.ssp370.Amon.gn',\n",
+       " 'AerChemMIP.BCC.BCC-ESM1.ssp370.LImon.gn']"
       ]
      },
-     "execution_count": 5,
+     "execution_count": 6,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "col.df.columns"
+    "col.keys()[:10]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Each key in this list points to all assets (files) that will be concatenated, merged into a single, high-level aggregated xarray dataset."
```

Comment:
"Each key in this list points to the assets (files) that will be merged into a single xarray dataset."